classes/class-wc-connect-help-provider.php

```diff
@@ -339,7 +417,23 @@ protected function get_form_schema() {
 						}
 					}
 
-					// TODO - support other types like toggles, textareas
+					if ( 'textarea' === $fieldsetitem->type ) {
+						$form_properties[ $fieldsetitem->key ] = array(
+							'title' => $fieldsetitem->title,
+							'type' => 'string', // textarea is a layout concept, not a schema concept
+							'description' => property_exists( $fieldsetitem, 'description' ) ? $fieldsetitem->description : '',
+						);
+					}
+
+					if ( 'boolean' === $fieldsetitem->type ) {
+						$form_properties[ $fieldsetitem->key ] = array(
+							'title' => $fieldsetitem->title,
+							'type' => 'boolean',
+							'text' => property_exists( $fieldsetitem, 'text' ) ? $fieldsetitem->text : '',
+							'description' => property_exists( $fieldsetitem, 'description' ) ? $fieldsetitem->description : '',
+							'saveontoggle' => property_exists( $fieldsetitem, 'saveontoggle' ) ? $fieldsetitem->saveontoggle : '',
```

Comment:
The default value for ```saveontoggle``` should be ```FALSE``` instead of an empty string. It doesn't really matter because Javascript will check for truthy values but still.